prepare-cluster-dev.sh

```diff
@@ -0,0 +1,40 @@
+
+#!/bin/bash
+
+# Exit if any of the intermediate steps fail
+set -e
+# set bash debug
+set -x
+
+# install OpenShift GitOps
+oc apply -k https://github.com/cloud-native-toolkit/deployer-cluster-prep//argocd-operator/?ref=develop --insecure-skip-tls-verify=true
```

Comment:
Change ref to main instead of develop

Reply:
the -dev is the prep cluster that tests the develop branch